src/Paramore.Brighter.Extensions.Hosting/TimedOutboxArchiver.cs

```diff
@@ -99,7 +99,7 @@ private async Task Archive(object state, CancellationToken cancellationToken)
                 {
                     IAmAnExternalBusService externalBusService = scope.ServiceProvider.GetService<IAmAnExternalBusService>();
                     
-                    await externalBusService.ArchiveAsync(options.MinimumAge,  new RequestContext(), cancellationToken);
+                    await externalBusService.ArchiveAsync(TimeSpan.FromHours(options.MinimumAge),  new RequestContext(), cancellationToken);
```

Comment:
Again, I think we change MinimumAge to be a TimeSpan

Reply:
Agreed, fixed